src/hooks.ts

```diff
@@ -413,5 +428,14 @@ export const useDecision: UseDecision = (flagKey, options = {}, overrides = {})
     return (): void => {};
   }, [optimizely.getIsReadyPromiseFulfilled(), options.autoUpdate, optimizely, flagKey, getCurrentDecision]);
 
+  useEffect(() => {
+    if (lastUserUpdate) {
+      setState(prevState => ({
+        ...prevState,
+        ...getCurrentDecision(),
+      }));
+    }
+  }, [lastUserUpdate]);
```

Comment:
This whole implementation for forced decision looks great. Can you add unit tests to cover all the scenarios